server/punishments.ts

```diff
@@ -1455,7 +1458,7 @@ export const Punishments = new class {
 
 				Punishments.autolock(user, 'staff', 'PunishmentMonitor', reason, message);
 				if (typeof user !== 'string') {
-					// tslint:disable-next-line: max-line-length
+					// eslint-disable-next-line max-len
 					(user as User).popup("|modal|You've been locked for breaking the rules in multiple chatrooms.\n\nIf you feel that your lock was unjustified, you can still PM staff members (%, @, &, and ~) to discuss it" + (Config.appealurl ? " or you can appeal:\n" + Config.appealurl : ".") + "\n\nYour lock will expire in a few days.");
```

Comment:
Honestly I would just give up and multiline this.
```suggestion
					(user as User).popup(
						`|modal|You've been locked for breaking the rules in multiple chatrooms.\n\n` +
						`If you feel that your lock was unjustified, you can still PM staff members (%, @, &, and ~) to discuss it${Config.appealurl ? " or you can appeal:\n" + Config.appealurl : "."}\n\n` +
						`Your lock will expire in a few days.`
					);
```